docs/elements.md

```diff
@@ -1665,6 +1688,12 @@ def generate(data):
 | `negative-weights`          | boolean | false              | Whether to recognize negative weights in an adjacency matrix. If set to false, then all weights at most 0 are ignored (not counted as an edge). If set to true, then all weights that are not `None` are recognized.                                                    |
 | `directed`                  | boolean | true               | Whether to treat edges in an adjacency matrix as directed or undirected. If set to false, then edges will be rendered as undirected. _The input adjacency matrix must be symmetric if this is set to false._                                                            |
 | `weights-presentation-type` | string  | `'f'`              | Number display format for the weights when using an adjacency matrix. If presentation-type is 'sigfig', each number is formatted using the to_precision module to digits significant figures. Otherwise, each number is formatted as `{:.{digits}{presentation-type}}`. |
+| `params-name-networkx`      | string  | `None`             | The name of a parameter containing a networkx graph to use as input for the graph.                                                                                                                                                                                      |
+| `networkx-edge-label`       | string  | `None`             | The key from the edge data dictionary to set to the `"label"` entry for display. By default, the entry stored in `"label"` will be displayed.                                                                                                                           |
```

Comment:
Is this what you were referring to here?

> I'm even wondering now whether the edge relabeling is needless abstraction.

If so, I agree; let's remove this unless there's a really compelling reason for this to be an attribute we support.

Reply:
Yes, I was referring to this edge label parameter. It's convenient in some cases but can be replaced by a two-liner on the side of the instructor.